pkg/cmd/server/server.go

```diff
@@ -75,11 +75,19 @@ const (
 	defaultMetricsAddress = ":8085"
 )
 
+type serverConfig struct {
+	pluginDir, metricsAddress                   string
+	backupSyncPeriod, podVolumeOperationTimeout time.Duration
+	restoreResourcePriorities                   []string
+	restoreOnly                                 bool
+}
+
 func NewCommand() *cobra.Command {
 	var (
 		logLevelFlag   = logging.LogLevelFlag(logrus.InfoLevel)
 		pluginDir      = "/plugins"
 		metricsAddress = defaultMetricsAddress
+		config         = serverConfig{}
```

Comment:
let's set the fields that have defaults to those values here, that way the default values will show up in the CLI help.  Still keep the `applyConfigDefaults` func since we don't want to allow zero values for the timeouts, etc.